Build fail with melodic armhf

### Commit
0.1.5

### Steps to reproduce
How can the issue/bug be reproduced?
1. Build on armhf system

### Expected behavior
Should be successful

### Observed behavior
```
07:57:06 /tmp/binarydeb/ros-melodic-psen-scan-v2-0.1.5/src/psen_scan_driver.cpp: In function ‘int main(int, char**)’:
07:57:06 /tmp/binarydeb/ros-melodic-psen-scan-v2-0.1.5/src/psen_scan_driver.cpp:81:33: error: ‘*((void*)& config_builder +4)’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
07:57:06      ScannerConfigurationBuilder config_builder;
07:57:06                                  ^~~~~~~~~~~~~~
07:57:06 cc1plus: all warnings being treated as errors
```

See https://build.ros.org/job/Mbin_ubhf_uBhf__psen_scan_v2__ubuntu_bionic_armhf__binary/21/console

--- a/src/psen_scan_driver.cpp
+++ b/src/psen_scan_driver.cpp
@@ -78,20 +78,20 @@ int main(int argc, char** argv)
                                                        getOptionalParamFromServer<double>(
                                                            pnh, PARAM_ANGLE_END, configuration::DEFAULT_ANGLE_END)) };
 
-    ScannerConfigurationBuilder config_builder;
-    config_builder
-        .hostIP(getOptionalParamFromServer<std::string>(pnh, PARAM_HOST_IP, configuration::DEFAULT_HOST_IP_STRING))
-        .hostDataPort(
-            getOptionalParamFromServer<int>(pnh, PARAM_HOST_DATA_PORT, configuration::DATA_PORT_OF_HOST_DEVICE))
-        .hostControlPort(
-            getOptionalParamFromServer<int>(pnh, PARAM_HOST_CONTROL_PORT, configuration::CONTROL_PORT_OF_HOST_DEVICE))
-        .scannerIp(getRequiredParamFromServer<std::string>(pnh, PARAM_SCANNER_IP))
-        .scannerDataPort(configuration::DATA_PORT_OF_SCANNER_DEVICE)
-        .scannerControlPort(configuration::CONTROL_PORT_OF_SCANNER_DEVICE)
-        .scanRange(scan_range)
-        .enableDiagnostics();
-
-    ScannerConfiguration scanner_configuration{ config_builder.build() };
+    ScannerConfiguration scanner_configuration{
+      ScannerConfigurationBuilder()
+          .hostIP(getOptionalParamFromServer<std::string>(pnh, PARAM_HOST_IP, configuration::DEFAULT_HOST_IP_STRING))
+          .hostDataPort(
+              getOptionalParamFromServer<int>(pnh, PARAM_HOST_DATA_PORT, configuration::DATA_PORT_OF_HOST_DEVICE))
+          .hostControlPort(
+              getOptionalParamFromServer<int>(pnh, PARAM_HOST_CONTROL_PORT, configuration::CONTROL_PORT_OF_HOST_DEVICE))
+          .scannerIp(getRequiredParamFromServer<std::string>(pnh, PARAM_SCANNER_IP))
+          .scannerDataPort(configuration::DATA_PORT_OF_SCANNER_DEVICE)
+          .scannerControlPort(configuration::CONTROL_PORT_OF_SCANNER_DEVICE)
+          .scanRange(scan_range)
+          .enableDiagnostics()
+          .build()
+    };
 
     ROSScannerNode ros_scanner_node(pnh,
                                     DEFAULT_PUBLISH_TOPIC,

